hydroflow/tests/surface_async.rs

```diff
@@ -263,3 +264,106 @@ pub async fn test_futures_stream_sink() -> Result<(), Box<dyn Error>> {
 
     Ok(())
 }
+
+#[tokio::test(flavor = "current_thread")]
+async fn asynctest_dest_sink_bounded_channel() {
+    // This is for testing. Don't actually use unbounded channels, as their input will end up
```

Comment:
Comment is a bit cryptic. If I understand right you mean:
"In ordinary code we would use unbounded channels, but that removes our ability to control buffering in a test. For this test, we restrict ourselves to a bounded channel (size 5)." I also don't understand what it says about `send.send(x).unwrap())`... is that a vestigial comment?